plugins/filter/tests/tests.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testdefinition version="1.0">
+  <suite name="voicecall-filter-plugin" domain="mw">
+    <set name="unit-tests" feature="voicecall-filter-plugin">
+       <case manual="false" name="tst_filter">
+         <step>HOME=$PWD /opt/tests/voicecall/plugins/filter/tst_filter</step>
```

Comment:
Wrong path.

Reply:
The HOME part seems to work if not in home directory :)

If you want to avoid using the home, perhaps it could be something hard-coded, e.g. HOME=/opt/tests/voicecall/filter ?

Reply:
Ah, sure. Corrected. I changed the suite and set names also.